args.js

```diff
@@ -32,7 +32,7 @@ module.exports = function parseArgs (args) {
       watch: false,
       debug: false,
       debugPort: 9320,
-      'ignore-watch': 'node_modules build dist .git bower_components logs .swp',
+      'ignore-watch': 'node_modules build dist .git bower_components logs .swp .nyc_output',
```

Comment:
watch tests hang without this because tap is watching this folder too without the ignore